File: c++/1176.cpp
#include <iostream>

using namespace std;

int main() {

    long long int fib[61], t, n, a, b, s;
    a = -1;
    b = 1;

    for(int j = 0; j <= 60; j++) {
        s = a + b;
        a = b;
        b = s;
        fib[j] = s;
    }

    cin >> t;

    for(int i = 1; i <= t; i++) {
        cin >> n;
        cout << "Fib(" << n << ") = " << fib[n] << endl;
    }

    return 0;
}

File: python3/1176.py
testes = int(input())

fib = []
a = -1
b = 1

for index in range(61):
    c = a + b
    fib.append(c)
    a = b
    b = c

for index in range(testes):
    entrada = int(input())
    print("Fib({}) = {}".format(entrada, fib[entrada]))
